website/content/docs/configure-hcp-packer.mdx

```diff
@@ -0,0 +1,88 @@
+
+** Required Environment Variables**
+
+- `HCP_CLIENT_ID`         - The HCP client ID of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_CLIENT_SECRET`    - The HCP client secret of the HashiCorp Cloud Platform service principle that will be authenticating Packer to the HCP Packer registry.
+
+- `HCP_PACKER_BUCKET_NAME` -  The name of the Image bucket where Packer will store information about each artifact from a build within the configured template. 
+
+** Optional Environment Variables**
+
+- `HCP_PACKER_BUILD_FINGERPRINT` -  A unique identifier assigned to a build run to determine if metadata for a template build on the registry is complete. A fingerprint can 
+not be more than 32 characters long.
+
+- `HCP_PACKER_REGISTRY`    -  Allows users to toggle HCP Packer publishing off. Only valid when all required variables have been set. Allowed values are [0|OFF].
+
+~> **Build Fingerprint**: By default, Packer uses the HEAD Git SHA for the template file to create a fingerprint. 
+If your template is not version controlled, you must set the HCP_PACKER_BUILD_FINGERPRINT environment variable and set it to a unique value each time you call packer build. 
+Packer will not build the template unless it detects a build fingerprint.
+
+## HCP Packer Registry HCL Block
+
+HCL2 templates, along with support for environment variables, can be configured using a `hcp_packer_registry` block, lets you customize the metadata Packer sends to HCP Packer Registry. It configures the details of an image that is created or updated within the HCP Packer registry. 
+
+The HCP Packer registry block in HCL is the preferred method for configuring Packer for the HCP Packer registry,
+as it allows for user customizable metadata. This block can not be used with legacy JSON template; nor is there a PACKER_CONFIG equivalent. 
+
+Usage of the `hcp_packer_registry` block requires that the environment variables `HPC_CLIENT_ID` and `HCP_CLIENT_SECRET` be set. The existence of any other variable such
+as `HCP_PACKER_BUCKET_NAME` or `HCP_PACKER_BUILD_FINGERPRINT` will override any configuration settings defined within the `hcp_packer_registry` block. 
+
+
+Below is an example of the `hcp_packer_registry` block. 
+```hcl
+source "happycloud" "macos" {
+  os = "macos_amd64"
+}
+
+build {
+   hcp_packer_registry {
+    bucket_name = "ios-dev"
+
+    description = " A description about the image which artifact is being published to HCP Packer Registry."
+
+    bucket_labels = {
+      "team" = "ios-development",
+      "os"   = "macos"
+    }
+
+    build_labels = {
+      "xcode"   = "11.3.0"
+      "version" = "Big Sur"
+    }
+  }
+
+  sources = ["source.happycloud.macos"]
+}
+
+```
+
+See [`hcp_packer_registry`](docs//templates/hcl_templates/blocks/build/hcp_packer_registry) for a full list configuration arguments.
```

Comment:
We always want to say "refer to" to avoid ableist language :) 
```suggestion
Refer to [`hcp_packer_registry`](docs//templates/hcl_templates/blocks/build/hcp_packer_registry) for a full list configuration arguments.
```

Reply:
Got it. Thank you for calling this out.